CONTRIBUTING.md

```diff
@@ -48,11 +47,7 @@ getting free help.
 
 ## Discuss your design
 
-The project welcomes submissions but please let everyone know what
-you're working on if you want to change or add something to the Gitea
-repositories.
-
-Before starting to write something new for the Gitea project, please [file
+The project welcomes submissions. But, if you want to change or add something to the Gitea repositories, please let everyone know what you're working on; before starting to write something new, [file
```

Comment:
No comments should be after the but.

Reply:
Also, the semicolon is unnecessary

Reply:
Yeah, a bit of a weird transition on this one. Replaced it with something a bit cleaner.